<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgFBLMidReg</class>
 <widget class="CItemDlg" name="CDlgFBLMidReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mouth Back Middle Region Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>CheckRegion:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnInCircle">
       <property name="text">
        <string>InnerCircle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOutCircle">
       <property name="text">
        <string>OutterCircle</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>82</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Edge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinEdge">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Area</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinArea">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkShadow">
       <property name="text">
        <string>ExcludeRingShadow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>±</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinShadowAng">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>89</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>°</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CItemDlg</class>
   <extends>QWidget</extends>
   <header>citemdlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinEdge</tabstop>
  <tabstop>spinArea</tabstop>
  <tabstop>btnOutCircle</tabstop>
  <tabstop>btnInCircle</tabstop>
  <tabstop>checkShadow</tabstop>
  <tabstop>spinShadowAng</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
 </slots>
</ui>
